samples/python/pbapi/__init__.py

```diff
@@ -747,6 +750,56 @@ def bulk_register(self, arguments):
                     print('[WARN] Unable to create beacon object: {}'.format(err.message))
                     continue
 
+    def set_property(self, arguments):
+        """
+        Adds (or replaces) properties on beacons.
+
+        Accepts a CSV file with a beacon_name column and additional columns whose header is the
+        property name and whose values on each row are the property values.
+
+        Args:
+            arguments: list of arguments passed from CLI. Pass ['--help'] for details.
+
+        Returns:
+            Nothing.
+        """
+        args_parser = argparse.ArgumentParser(description='Sets a property on a beacon.',
```

Comment:
"Sets properties on beacons based on the content of a CSV files"?

Reply:
Done.